README.md

```diff
@@ -110,13 +110,15 @@ for detailed explanation and flags of each command see help
 | endpoint.eth.proxyEndpoints.disabledEndpoints | ENDPOINT_ETH_DISABLEDENDPOINTS                          | empty list                                   | ✅              | e.g.: debug_traceTransaction                                                                                                                        |
-| rpcNode.geth.HTTPHost                         | RPCNODE_GETH_HTTPHOST                                   | `localhost`                                  | ❌              |                                                                                                                                                     |
-| rpcNode.geth.HTTPPort                         | RPCNODE_GETH_HTTPPORT                                   | `8545`                                       | ❌              |                                                                                                                                                     |
-| rpcNode.geth.HTTPVirtualHosts                 | RPCNODE_GETH_HTTPVIRTUALHOSTS                           | `["localhost"]`                              | ❌              |                                                                                                                                                     |
-| rpcNode.geth.HTTPModules                      | RPCNODE_GETH_HTTPMODULES                                | `["net", "web3", "eth", "parity"]`           | ❌              |                                                                                                                                                     |
-| rpcNode.geth.WSHost                           | RPCNODE_GETH_WSHOST                                     | `localhost`                                  | ❌              |                                                                                                                                                     |
-| rpcNode.geth.WSPort                           | RPCNODE_GETH_WSPORT                                     | `8545`                                       | ❌              |                                                                                                                                                     |
-| rpcNode.geth.WSModules                        | RPCNODE_GETH_WSMODULES                                  | `["net", "web3", "eth", "parity"]`           | ❌              |                                                                                                                                                     |
+| rpcNode.HTTPHost                              | RPCNODE_HTTPHOST                                        | `localhost`                                  | ❌              |                                                                                                                                                     |
```

Comment:
`rpcNode.HTTPHost` should be `rpcNode.httpHost`
please also fix rest of the new configs as well